python/orchestrate/_internal/convert.py

```diff
@@ -122,6 +122,8 @@ def cda_to_fhir_r4(
         patient_id: Optional[str] = None,
         patient_identifier: Optional[str] = None,
         patient_identifier_system: Optional[str] = None,
+        include_cda_in_the_output: Optional[bool] = None,
```

Comment:
I made comments about these param names in the Hendrix PR - need to sort those out and propagate any changes here.